### PR TITLE
Makes jellypeople look like Slimepeople instead of Stargazers

### PR DESCRIPTION
**Pull request by tiramisuapimancer**
## About The Pull Request
Gives "jelly" race all slimepeople subspecies are derived from the sprites of their Slime subtype (the splitters) instead of the ugly Stargazer ones, gives them hair and beards. Incidentally makes them available by default in the config but this was something I was too lazy to care about, it's only like this because  I made them roundstart in testing by changing that. Doesn't matter because it's not the actual config used. I will remove it if asked to

This also, incidentally, makes all slime types have hair, not just splitters. This was left in on purpose

## Why It's Good For The Game
Makes the default Jelly people race ready for round start selection (IMO no balance changes are needed) by giving them a more likeable appearance. Makes people on the Discord shut up about slimepeople. Makes two of the xenobio slime types suck less

## Changelog
:cl:
tweak: Jelly species uses Slime sprites instead of Stargazer
tweak: Jelly people and all their subspecies now have hair
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
Good for you, you actually coded this in instead of just asking someone else to do it

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 1]
```diff
@@ -1,11 +1,13 @@
 /datum/species/jelly
-	// Entirely alien beings that seem to be made entirely out of gel. They have three eyes and a skeleton visible within them.
+	// Entirely alien beings that seem to be made entirely out of gel. They have three eyes and a skeleton visible within them. actually they dont any more lol
 	name = "Jellyperson"
 	id = "jelly"
 	default_color = "00FF90"
 	say_mod = "chirps"
-	species_traits = list(MUTCOLORS,EYECOLOR,NOBLOOD)
+	species_traits = list(MUTCOLORS,EYECOLOR,HAIR,FACEHAIR,NOBLOOD)//(MUTCOLORS,EYECOLOR,NOBLOOD)
```
Comment (Kmc2000) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm:7]:
```suggestion
	species_traits = list(MUTCOLORS,EYECOLOR,HAIR,FACEHAIR,NOBLOOD)//(MUTCOLORS,EYECOLOR,NOBLOOD) //Nsv13
```

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 	inherent_traits = list(TRAIT_TOXINLOVER)
+	hair_color = "mutcolor"
```
Comment (Kmc2000) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm:9]:
```suggestion
	hair_color = "mutcolor" //Nsv13
```

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+	hair_alpha = 150
```
Comment (Kmc2000) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm:10]:
```suggestion
	hair_alpha = 150 //Nsv13
```

**Review by Kmc2000 — CHANGES_REQUESTED** [review]

File: config/game_options.txt [hunk 4]
```diff
@@ -468,7 +468,7 @@ ROUNDSTART_RACES ipc
 ROUNDSTART_RACES ethereal
 ROUNDSTART_RACES apid
 ROUNDSTART_RACES synthetic
-#ROUNDSTART_RACES jelly
+ROUNDSTART_RACES jelly
```
Comment (Karmic-Skink) [inline config/game_options.txt:471]: config changes like these tend not to actually make it onto the live server and need to be done per hostbox

Comment (tiramisuapimancer) [reply config/game_options.txt:471]: Acknowledged that in the thing but I'll change it anyhow, this is pretty much completely inconsequential nomatter which way it goes

Comment (tiramisuapimancer) [reply config/game_options.txt]: there it goes

**Comment by tiramisuapimancer** [comment]
im not a hundred percent sure how github works but   some things were done so yes

**Comment by theboy6545** [comment]
speedmerge

**Comment by tiramisuapimancer** [comment]
hw do i make it stop saying changes were requested. i addressed the requested changes, ree

**Pull request closed without merging**